### PR TITLE
Enable Light Baking Cookies by default + Warning

### PR DESCRIPTION
**Pull request by skhiat**
Update Light Baking with Cookie

* If Project Settings > disable light cookies in baking && project use baking or mixed light with cookie show warning message
* For new project, enable light baking cookies by default.

Reference PR:
https://github.com/Unity-Technologies/ScriptableRenderPipeline/pull/6362

### **Please read**
**PR workflow guidelines**
* SRP ABV will start automatically on Yamato when you open your PR
* Changes to docs and md files will **not** trigger ABV jobs 
* Consider making use of **draft PRs** if you are not 100% sure that your PR is ready for review
* ABV will restart if you add a new commit to a branch with an open PR (hence why you should consider using draft PRs)
* Adding [skip ci] (case insensitive) to the title of PRs will stop any jobs being trigger automatically - you will need to open Yamato and find your branch to run ABV
* You can also add [skip ci] to commit messages to prevent CI from running on that push
* Add [cancel old ci] to your commit message if you've made changes you want to test and no longer need the previous jobs

### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [ ] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [ ] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR.
- [ ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

---
### Testing status

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.Skin.cs [hunk 1]
```diff
@@ -42,6 +42,7 @@ sealed class Styles
             public readonly GUIContent cookieTextureTypeError = new GUIContent("HDRP does not support the Cookie Texture type, only Default is supported.", EditorGUIUtility.IconContent("console.warnicon").image);
             public readonly string cookieNonPOT = "HDRP does not support non power of two cookie textures.";
             public readonly string cookieTooSmall = "Min texture size for cookies is 2x2 pixels.";
+            public readonly string cookieBaking = "Light Baking for cookies disabled on the Project Settings, .";
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.Skin.cs:45]: Typo with the coma at the end?

Comment (skhiat) [reply com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.Skin.cs:45]: [Fixed]

**Comment by sebastienlagarde** [comment]
This PR come with this commit to fix compil issue in editor: 
Fix compil issue with Enable BakeCookie - 6cdf38e5dc100bb9c370be94f5c9d73de2f19aef

**Comment by sebastienlagarde** [comment]
+commit: Enable Light Baking Cookies by default + Warning part 3 #79

**Pull request merged**